### ISSUE TITLE
microservice: post readme and swagger file to db

### ISSUE THREAD

**Issue by sbtaylor15** [ortelius/ortelius#327]
Need to associate the readme file into the db for rendering later on in the UI. 

Base64 encode the file and associate it to the component version.  dm.dm_textfile will be the table used to store the file.  base64 encoded files break on character 76 so multiple lines per file in the table.  filetype should be set to "readme".  The dm_textfile will also be used for the swagger file.

CREATE TABLE IF NOT EXISTS dm_textfile
(
 compid integer primary key,
 filetype varchar(20),
 lineno integer,
 base64str varchar(80)
);

**Comment by sbtaylor15** [comment]
Microservice to use as a reference - https://github.com/ortelius/ortelius-ms-compitem-crud

**Comment by sbtaylor15** [comment]
Json payload from client

```
{
    compid: 123,
    filetype: "readme",
    file: [
        "aaaaaddd",
        "dddfklkl"
    ]
}
```

**Comment by sbtaylor15** [comment]
Handle "swagger" filetype as well

**Comment by sbtaylor15** [comment]
[ortelius-ms-textfile-crud](https://github.com/ortelius/ortelius-ms-textfile-crud) has the code.  Need to add in the Dockerfile and cloudbuild.yaml to the repo

**Comment by sbtaylor15** [comment]
Resolves ortelius/ortelius-ms-textfile-crud#3
